src/Microsoft.Health.Fhir.TemplateManagement/Client/OrasCacheClient.cs

```diff
@@ -0,0 +1,43 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.ContainerRegistry.Models;
+using Microsoft.Health.Fhir.TemplateManagement.Models;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.TemplateManagement.Client
```

Comment:
Will this implement the `IOciClient` interface?

Reply:
Changed it to partial  orasclient  class.